examples/extend-graphql-schema-nexus/schema.ts

```diff
@@ -94,7 +94,7 @@ export function extendGraphqlSchema(baseSchema: GraphQLSchema) {
     },
 
     // Typescript output settings, probably something you might commit in dev
-    shouldGenerateArtifacts: process.env.NODE_ENV !== 'production',
+    shouldGenerateArtifacts: false ?? process.env.NODE_ENV !== 'production',
```

Comment:
I think `false ?? process.env.NODE_ENV !== 'production',` will always evaluate to false.  The nullish coalescing operator only detects `null` or `undefined`.

Reply:
sorry, this is a work-around for a bug in that particular example that needs to be fixed